### PR TITLE
Set up test environment for Client

### PR DESCRIPTION
**Pull request by swardbladpaulo**
[Set up test environment for Client](https://www.pivotaltracker.com/story/show/176033308)
### User Story
```
As a development team,
In order to unit test our code
We would like to set up Cypress
```
## Changes proposed in this pull request:
* add enzyme, setupTests.js
* installed cypress, change baseUrl

## What I have learned working on this feature:
* Don't go too fast when installing cypress

## Packages/Gems added
* Cypress
* Enzyme

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (emtalen) [inline cypress/fixtures/example.json]: you can remove this file

File: cypress/fixtures/profile.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": 8739,
+  "name": "Jane",
+  "email": "jane@example.com"
+}
```
Comment (emtalen) [inline cypress/fixtures/profile.json]: same with this

File: cypress/fixtures/users.json [hunk 3]
```diff
@@ -0,0 +1,232 @@
+[
```
Comment (emtalen) [inline cypress/fixtures/users.json]: aaand this

File: package.json [hunk 4]
```diff
@@ -34,5 +38,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "cypress": "^6.1.0",
```
Comment (emtalen) [inline package.json]: I asked in the camp channel that you down grade this to 5.6.0, please do

**Review by emtalen — CHANGES_REQUESTED** [review]

File: cypress.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3001"
```
Comment (tochman) [inline cypress.json]: I would add the security setting from start.

File: package.json (hunk 4 above)
Comment (tochman) [reply package.json]: Yes, the mocking of server responses are not really up to speed in this version. Please downgrade.

File: cypress/fixtures/example.json (hunk 1 above)
Comment (swardbladpaulo) [reply cypress/fixtures/example.json]: Deleted this file

File: cypress/fixtures/profile.json (hunk 2 above)
Comment (swardbladpaulo) [reply cypress/fixtures/profile.json]: Deleted this file

File: cypress/fixtures/users.json (hunk 3 above)
Comment (swardbladpaulo) [reply cypress/fixtures/users.json]: Deleted this file

File: package.json (hunk 4 above)
Comment (swardbladpaulo) [reply package.json]: Bumped down to this version.

File: cypress.json (hunk 5 above)
Comment (swardbladpaulo) [reply cypress.json]: Noted and added.

**Review by emtalen — APPROVED** [review]

**Pull request merged**